scripts/quests/otherAreas/Rycharde_the_Chef.lua

```diff
@@ -0,0 +1,142 @@
+-----------------------------------
+-- Rycharde the Chef
+-----------------------------------
```

Comment:
Can we have the big block that lists all the involved NPCs with their `!pos` commands here please?
https://github.com/DerpyProjectGroup/topaz/pull/63/files#diff-8d167d9c9b0765dcc7219c1b8f94daf4f7f02fc7a8866b68fb4158fda676f0eaR1-R11

Reply:
On it